src/clj/puppetlabs/puppetserver/certificate_authority.clj

```diff
@@ -69,7 +69,19 @@
    :ruby-load-path           [schema/Str]
    :gem-path                 schema/Str
    :signeddir                schema/Str
-   :serial                   schema/Str})
+   :serial                   schema/Str
+   ;; This would be list of compile master node names (FQDNs) including MoM
```

Comment:
I believe these need to be certnames, not node names, or FQDNs (though we often combine those concepts).